src/pieces/commands/list_command.py

```diff
@@ -25,20 +27,24 @@ def list_command(cls, **kwargs):
 
     @classmethod
     @check_assets_existence
-    def list_assets(cls, max_assets: int = 10,**kwargs):
-        assets = kwargs.get("assets",Settings.pieces_client.assets()[:max_assets])
-
-        select_menu = PiecesSelectMenu(
-            [
-                (f"{i}: {asset.name}", {"asset_id":asset.id,**kwargs})
-                for i, asset in enumerate(assets,start=1)
-            ],
-             AssetsCommands.open_asset,kwargs.get("footer"))
+    def list_assets(cls, **kwargs):
+        assets = kwargs.get("assets",[BasicAsset(item.id) for item in BasicAsset.get_identifiers()])
+
+        select_menu = PiecesSelectMenu([], AssetsCommands.open_asset,kwargs.get("footer"))
+        def update_assets():
+            for i,asset in enumerate(assets,start=1):
+                select_menu.add_entry(
+                    (f"{i}: {asset.name}", {"asset_id":asset.id,**kwargs}))
+
+        threading.Thread(target=update_assets).start()
         select_menu.run()
+        
+
+        
 
     @classmethod
     def list_models(cls):
-        models = [(f"{idx}: {model_name}", {"MODEL_INDEX":idx,"show_warning":False}) 
+        models = [(f"{idx}: {model_name}", {"MODEL_INDEX":idx}) 
```

Comment:
any reason you wanted to pull the show_warning?

Reply:
It is not useful since we already removed the deprecated commands.